rust/src/action/parquet_read/mod.rs

```diff
@@ -173,14 +197,19 @@ impl Add {
                         "nullCount" => match record.get_group(i) {
                             Ok(row) => {
                                 for (name, field) in row.get_column_iter() {
-                                    match field.try_into() {
-                                        Ok(count) => {
-                                            stats.null_count.insert(name.clone(), count);
-                                        },
+                                    match field {
+                                        Field::Null => {}
                                         _ => {
-                                            log::warn!("Expect type of nullCount field to be struct or int64, got: {}", field);
-                                        },
-                                    };
+                                            match field.try_into() {
```

Comment:
huh I'm unsure how the compiler knows what the destination types is here, but I guess it's ColumnCountStat?